server/routes/cohorts.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express')
+
+const db = require('../db/cohorts')
+
+const router = express.Router()
+
+router.get('/', (req, res) => {
+  db.getCohorts()
+    .then(cohorts => {
+      res.render('index', { cohorts: cohorts })
```

Comment:
Looks like you're trying to render something here? - I think we need to send back a response rather than render anything.